static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -54,6 +54,41 @@
     position: relative;
 }
 
+.youtube-media-atom__paid-for-overlay {
+    background-color: $brightness-100;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    height: 85px;
+    opacity: .5;
+}
+
+.youtube-media-atom__paid-for-label {
+    font-size: 16px;
+    color: #000000;
```

Comment:
We should user the variable name for this hex code too. https://github.com/guardian/frontend/pull/21685#discussion_r306846091

Reply:
I have already checked there's no var for #000000

Reply:
😆 yet we have one for #FFFFFF?! OK well I'll mark this as resolved!